### ISSUE TITLE
propose obsoletion phosphoprotein binding terms

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#20128]
GO:0106153    phosphorylated histone binding (3 EXP) (2 PomBase)
GO:0051219    phosphoprotein binding  (206 EXP)
GO:0045309    protein phosphorylated amino acid binding

These should be captured with a modified form in column ~17~ 16

Modified forms for col~17~ 16 can be requested here
https://github.com/PROconsortium/PRoteinOntology/issues

**Comment by mah11** [comment]
No, these GO terms are defined as binding TO a phosphorylated protein, so the PRO ID should go in an extension (GAF column 16). Column 17 is for modified forms that DO the action.

(I do agree that using PRO terms in extensions makes the GO terms unnecessary in the ontology.)

**Comment by ValWood** [comment]
Good point. I did mean 16. I still get these mixed up.

I did see how these ad been used slightly differently

phosphorylated histone binding means that you bind *directly* to the phosphorylated residue
phosphoprotein binding means binding to a phosphoprotein, but not necessarily the phosphorylated residue, it is just a requirement that the residue be phosphorylated. It's an important distinction to model 
simultaneous or competitive binding, but this also  doesn't belong in GO (possibly a specific type of binding relationship?, this is probably how we will do this locally at PomBase)
